lib/pageflow/configuration.rb

```diff
@@ -252,6 +252,11 @@ class Configuration
     # take. Defaults to `[:captions, :subtitles, :descriptions]`.
     attr_reader :available_text_track_kinds
 
+    # Allow one user to be member of multiple accounts. Defaults to
+    # true.
+    # @since 12.1
```

Comment:
Please use `@since edge`. Tags will automatically be rewritten once a Pageflow version is released. That way there is no need to guess which minor version the feature will be in.